.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Report a bug in MegaQC
+title: "Bug report"
+labels: "bug"
+assignees: ""
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+
+1. Run megaqc with command: '...'
+2. Select: '...'
+3. See error
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+<!-- If required to reproduce
+**System [please complete the following information]:**
```

Comment:
Maybe combine this previous line into:
> System [please complete the following information, if relevant to the issue]

Reply:
Done. I should have read what I even copied there :)